Resolver/todoResolver.js

```diff
@@ -1,84 +1,66 @@
 
 const Todo = require("../models/todoModel");
 
 const TODO_ADDED = 'TODO_ADDED';
 
 const resolver = {
-    Subscription:{
-        notifyUsers: {
-            subscribe: () => pubsub.asyncIterator([TODO_ADDED])
-        }
-    },
-  todos: () => {
-    return Todo.find()
-      .then((todos) => {
-        return todos.map((todo) => {
-          return { ...todo._doc, _id: todo.id };
-        });
-      })
-      .catch((err) => {
-        throw err;
+  Query: {
+    todos: () => {
+      return Todo.findAll().then((todos) => {
+          return todos.map((todo) => {
+              return { ...todo.dataValues };
+          });
       });
   },
-  todo(args) {
-    return Todo.findById(args.todoId)
-      .then((todo) => {
-        return { ...todo._doc, _id: todo.id };
-      })
-      .catch((err) => {
-        throw err;
-      });
   },
-  createtodo: (args) => {
-    const todo = new Todo({
-      Todoitem: args.todoItem,
-      complete: false
-    });
-    return todo
-      .save()
-      .then((result) => {
-        var Message = { message: 'New todo has been added', todo: todo};
-        pubsub.publish(TODO_ADDED, { notifyUsers: Message });
-        return Todo.find()
-            .then((todos) => {
-                return todos.map((todo) => {
-                    return { ...todo._doc, _id: todo.id };
-                });
-            }).catch((err) => {
-                throw err;
-            });
-      })
-      .catch((err) => {
-        throw err;
+  Mutation: {
+    createTodo: async (context,args) => {
+      const todo = await Todo.create({
+        _id: Math.random().toString(),
+        Todoitem: args.input,
+        complete: false,
       });
-  },
-  updatetodo: (args) => {
-    return Todo.find({})
-        .then((todos) => {
-            const todo = todos.find((todo) => todo.id === args.todoid);
-            todo.complete = !todo.complete;
-            todo.save();
-            return todos.map((todo) => {
-                return { ...todo._doc, _id: todo.id };
-            });
-        })
-        
-  },
-  deletetodo: (args) => {
-    return Todo.find({})
-        .then((todos) => {
-            const todo = todos.find((todo) => todo.id === args.todoid);
-            todos.splice(todos.indexOf(todo), 1);
-            todo.delete()
-            return todos.map((todo) => {
-                return { ...todo._doc, _id: todo.id };
-            });
-        }).catch((err) => {
-            throw err;
+      const todos = await Todo.findAll();
+      pubsub.publish(TODO_ADDED, { todoAdded: todos });
+      return Todo.findAll().then((todos) => {
+        return todos.map((todo) => {
+          return { ...todo.dataValues };
+        });
+      });
+    },
+    updateTodo: async (context, args) => {
+      const todo = await Todo.findByPk(args.id);
+      todo.complete = !todo.complete;
+      await todo.save();
+      const todos = await Todo.findAll();
+      pubsub.publish(TODO_ADDED, { todoAdded: todos });
```

Comment:
make these function async to use await that will improve the readability of the code